cli/pkg/kctrl/cmd/app/delete.go

```diff
@@ -57,6 +57,7 @@ func NewDeleteCmd(o *DeleteOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 		AllowDisableWait: true,
 		DefaultInterval:  1 * time.Second,
 		DefaultTimeout:   5 * time.Minute,
+		WaitByDefault:    true,
```

Comment:
We should be using `o.pkgCmdTreeOpts.WaitByDefault`, right?

Reply:
For this one and the next we will always tail, these are the app commands